lib/driver.js

```diff
@@ -346,7 +366,7 @@ class XCUITestDriver extends BaseDriver {
   }
 
   async startWda (sessionId, realDevice) {
-    this.wda = new WebDriverAgent(this.xcodeVersion, this.opts);
+    this.wda = new WebDriverAgent(this.xcodeVersion, Object.assign(this.opts, {isFirstSession: this.isFirstSession}));
```

Comment:
I would prefer this be done right after we figure out if it is the first session.